code/go/0chain.net/smartcontract/minersc/handler.go

```diff
@@ -406,10 +406,16 @@ type nodeStat struct {
 //	484:
 func (mrh *MinerRestHandler) testNodeStat(w http.ResponseWriter, r *http.Request) {
 	var (
-		id               = r.URL.Query().Get("id")
+		provider_id      = r.URL.Query().Get("provider_id")
 		includeDelegates = strings.ToLower(r.URL.Query().Get("include_delegates")) == "true"
```

Comment:
Use camel format, i.e `providerID`

Reply:
we used to use snake case for url parameters, no?